src/lib/web/endpoints.rs

```diff
@@ -81,6 +83,12 @@ pub async fn mavlink(path: Option<Path<String>>) -> impl IntoResponse {
     crate::drivers::rest::data::messages(&path)
 }
 
+pub async fn post_mavlink(State(state): State<AppState>, message: String) {
+    if let Err(error) = state.message_tx.send(message) {
+        error!("Failed to send message to main loop: {error:?}");
+    }
+}
```

Comment:
maybe we want to have a "sender ID" arg to identify who is sending this message?

Reply:
I have added a debug message showing the client and message, if the idea was to use the stats API, it need some rework since multiple posts and multiple websockets will have different clients, and we can't create a stats for each one of them.